tests-beta/android/MASVS-NETWORK/MASTG-TEST-0x22-3.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Missing Certificate Pinning in Network Traffic 
+platform: network
+id: MASTG-TEST-0x22-3
+type: [static]
+weakness: MASWE-0047
+---
+
+## Overview
+
+There are various ways how [certificate pinning]("../../../Document/0x05g-Testing-Network-Communication.md#certificate-pinning-without-android-network-security-configuration") can be done for an application.
```

Comment:
```suggestion
```

If we remove it in the document, let's also remove the reference here.

Reply:
I think the text itself does make sense, but if the link is gone, it makes sense to remove that.